LOCALPORTALREHOMING.mkd

```diff
@@ -0,0 +1,142 @@
+# Portal rehoming environment
```

Comment:
Can we call this file running-local-portal.md

Reply:
Done